src/Calendar.jsx

```diff
@@ -349,6 +349,7 @@ let Calendar = React.createClass({
     )
 
     let ToolbarToRender = components.toolbar || Toolbar
+    const now = new Date()
```

Comment:
what would be the advantage to exposing this prop?  this should be an implementation detail